canonical-web-and-design/ubuntu.com
# Close CUBE beta

## Done

- Update copy on /cube to close the beta
- Redirect /cube/microcerts and /cube/study to /cube
- Update the CUBE contact form

## QA

- Open the demo at https://ubuntu-com-11224.demos.haus/cube
  - Verify that the content is correct according to https://github.com/canonical-web-and-design/commercial-squad/issues/491
- Click the "Sign up" button and verify that the form content is correct according to https://github.com/canonical-web-and-design/commercial-squad/issues/491
- Go to https://ubuntu-com-11224.demos.haus/cube/microcerts and verify that you are redirected to https://ubuntu-com-11224.demos.haus/cube
- Go to https://ubuntu-com-11224.demos.haus/cube/study and verify that you are redirected to https://ubuntu-com-11224.demos.haus/cube


## Issue

Fixes: https://github.com/canonical-web-and-design/commercial-squad/issues/491 // [[Jira](https://warthogs.atlassian.net/jira/software/projects/CUBE/boards/490?assignee=613a0f86ee2fd0006ae0c8bf&selectedIssue=CUBE-84)]

## Screenshots
